Update cli-reqs.md (#1352)
ARMmbed/Handbook: docs/tools/CLI/cli-setup/cli-reqs.md

1. Clone the Mbed CLI repository: `git clone https://github.com/ARMmbed/mbed-cli`.
1. Navigate to the `mbed-cli/tools/bash_completion` directory.

> 1. Copy the `mbed` script into the  `~/.bash_completion.d` directory (you may need to create this directory first, and if that is the case then you may also need to add a line like `source ~/.bash_completion.d/mbed` to `~/.bashrc`).

Reviewer: ```suggestion
1. Copy the `mbed` script into the  `~/.bash_completion.d` directory (you may need to create this directory first, and then you may also need to add a line, for example `source ~/.bash_completion.d/mbed` to `~/.bashrc`).
```
